docs/03-pages/01-building-your-application/06-configuring/13-debugging.mdx

```diff
@@ -98,6 +98,21 @@ Once the server starts, open a new tab in Chrome and visit `chrome://inspect`, w
 
 Debugging server-side code here works much like debugging client-side code with Chrome DevTools, except that when you search for files here with `Ctrl+P` or `⌘+P`, your source files will have paths starting with `webpack://{application-name}/./` (where `{application-name}` will be replaced with the name of your application according to your `package.json` file).
 
+### Inspect Server Errors with Chrome DevTools
+
+When you bumped to an error, inspecting the source code will be super helpful to trace the root cause of errors.
+In the incoming Next.js v15, Next.js will display a Node.js logo like green button on dev overlay during the inspect mode. By clicking that button you can copy the Chrome DevTool url into clipboard, and you can open a new browser tab with that url to inspect the Next.js server process with Chrome DevTool.
+
+There's a limitation of directly opening the DevTool in user-land pages, you'll have to click that button to copy the url and paste it in a new window or tab to inspect.
```

Comment:
I don't think that's needed. It's really a limitation for the implementation. If it didn't exist, we'd replace the copy button with a link

Reply:
Users might ask why we cannot open it directly in a window, since "it's just an URL like others". It's a pretty indirect way to start inspector. Might be nice to explain it somewhere, I didn't find a good place to do it on UI so I explain it here.

Reply:
Do we generally document implementation choices? If you feel strongly about it, we should avoid "user-land pages". I don't think that's a widely used term. Just say it can't be a link and you need to manually paste the url into the browser navigation bar.

Reply:
We're not always documenting those, I'll just drop this line